src/AutoUI/Actions/Update.tsx

```diff
@@ -87,38 +142,60 @@ export const Update = <T extends AutoUIBaseResource<T>>({
 			</Box>
 		);
 	}
+	const groupedActions = React.useMemo(
+		() => groupBy(updateActions, (action) => action.section),
+		[updateActions],
+	);
+	const actionHandlers = React.useMemo(
+		() =>
+			map(groupedActions, (actions) =>
+				actions.map((action) => {
+					const disabledReason =
+						autoUIGetDisabledReason(
+							selected,
+							hasOngoingAction,
+							action.type as 'update' | 'delete',
+							t,
+						) ?? disabledReasonsByAction[action.title];
 
-	const groupedActions = groupBy(updateActions, (action) => action.section);
-	const actionHandlers = map(groupedActions, (actions) =>
-		actions.map((action) => {
-			const disabledReason =
-				autoUIGetDisabledReason(
-					selected,
-					hasOngoingAction,
-					action.type as 'update' | 'delete',
-					t,
-				) ?? action.isDisabled?.({ affectedEntries: selected });
-
-			return {
-				content: action.title,
-				onClick: () =>
-					onActionTriggered({
-						action,
-						schema:
-							action.type === 'delete'
-								? {}
-								: autoUIJsonSchemaPick(
-										model.schema,
-										model.permissions[action.type],
-								  ),
-						affectedEntries: selected,
-					}),
-				tooltip:
-					typeof disabledReason === 'string' ? disabledReason : undefined,
-				disabled: !!disabledReason,
-				danger: action.isDangerous,
-			};
-		}),
+					return {
+						content: (
+							<DropdownItemContent<any>
```

Comment:
Cant' this be
```suggestion
							<DropdownItemContent<T>
```
?
Not a blocker though.

Reply:
I forgot to change `DropDownItemContentProps` after some tests... thank you :)